mitosheet/mitosheet/step_performers/replace.py

```diff
@@ -40,48 +40,57 @@ def execute(cls, prev_state: State, params: Dict[str, Any]) -> Tuple[State, Opti
         sheet_index: int = get_param(params, 'sheet_index')
         search_value: str = get_param(params, 'search_value')
         replace_value: str = get_param(params, 'replace_value')
-        
+        column_ids: List[ColumnID] = get_param(params, 'column_ids')
 
         # We make a new state to modify it
         post_state = prev_state.copy() 
 
         pandas_start_time = perf_counter()
-        
         df = post_state.dfs[sheet_index]
 
+        # If no column_ids are specified, then we replace the values in all columns
+        if column_ids is None or len(column_ids) == 0:
+            column_ids = df.columns
+
+        # Selected columns is the dataframe with only the columns we want to replace values in
+        selected_columns = df[column_ids]
```

Comment:
We shouldn't be accessing the dataframe by the column ids. I think we need to use the column ids map to get column headers, and then access by column headers here. Def make sure to rename variables through this step performer (as well as the code chunk) approproatiely as well.

Can you write one test where you create a mitosheet, rename a column, then replace in that column? Should catch this bug.